plugins/source/oracledb/resources/plugin/plugin_test.go

```diff
@@ -47,6 +47,7 @@ func createTable(ctx context.Context, db *sql.DB, table *schema.Table) error {
 	builder.WriteString("CREATE TABLE ")
 	builder.WriteString(client.Identifier(table.Name))
 	builder.WriteString(" (\n  ")
+	pk := make([]string, 0, len(table.PrimaryKeys()))
```

Comment:
Do we still need this change if we use the old test data code?

Reply:
Yes (as the gendata will gen the row data, but we still have the new table schema with multiple PK columns)

Reply:
Update after syncing on Slack. This is needed since we still use a new version of the schema that has multiple PK supoprt